frontend/dockerfile/docs/reference.md

```diff
@@ -1575,8 +1575,7 @@ conditions for cache reuse.
 
 > **Note**
 >
-> Available in [`docker/dockerfile-upstream:master-labs`](#syntax).
-> Will be included in `docker/dockerfile:1.7-labs`.
+> Not yet available in stable syntax, use [`docker/dockerfile:1-labs`](#syntax) version.
```

Comment:
This has not technically in `1-labs` yet. I think we can leave it to `1.7-labs`. Same for the rest.

Reply:
fixed